File: src/main/java/com/aa/mtg/game/turn/phases/PhaseUtils.java
package com.aa.mtg.game.turn.phases;

import static com.aa.mtg.game.turn.phases.Main1Phase.M1;
import static com.aa.mtg.game.turn.phases.Main2Phase.M2;

public class PhaseUtils {
    public static boolean isMainPhase(String phase) {
        return phase.equals(M1) || phase.equals(M2);
    }
}

File: game/src/main/resources/app/src/game/Turn/PhaseUtils.js
export class PhaseUtils {
  static getPhases() {
    return ['UT', 'UP', 'DR', 'M1', 'BC', 'DA', 'DB', 'AB', 'FS', 'CD', 'EC', 'M2', 'ET', 'CL']
  }

  static isMainPhase(phase) {
    return phase === 'M1' || phase === 'M2'
  }
}